test/fixtures/oidc_fixtures.dart

```diff
@@ -0,0 +1,35 @@
+
+import 'package:model/oidc/oidc_configuration.dart';
+import 'package:model/oidc/token_id.dart';
+import 'package:model/oidc/token_oidc.dart';
+
+class OIDCFixtures {
+  static final tokenOidcExpired = TokenOIDC(
+    'dab123',
+    TokenId('dab123'),
+    'dab456',
+    expiredTime: DateTime.tryParse('2024-01-22 10:00:00.000'));
+
+  static final tokenOidcWithRefreshTokenEmpty = TokenOIDC(
+    'dab123',
+    TokenId('dab123'),
+    '',
+    expiredTime: DateTime.tryParse('2024-01-22 10:00:00.000'));
+
+  static final tokenOidcWithTokenEmpty = TokenOIDC(
+    '',
+    TokenId('dab123'),
+    'dab456',
+    expiredTime: DateTime.tryParse('2024-01-22 10:00:00.000'));
+
+  static final tokenOidc = TokenOIDC(
+    'test123',
+    TokenId('test123'),
+    'test456',
+    expiredTime: DateTime.tryParse('2024-01-24 10:00:00.000'));
```

Comment:
Must be `DateTime.now() + 2 day`. If not, still failed if test run after 25-01-2024

Reply:
done